src/Bridge/Symfony/Messenger/DataPersister.php

```diff
@@ -98,4 +86,40 @@ public function remove($data, array $context = [])
                 ->with(new RemoveStamp())
         );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function loop($data, array $context = []): bool
+    {
+        try {
+            $value = $this->getMessengerAttributeValue($this->resourceMetadataFactory->create($context['resource_class'] ?? $this->getObjectClass($data)), $context);
+        } catch (ResourceClassNotFoundException $exception) {
+            return false;
+        }
+
+        return 'persist' === $value || (\is_array($value) && \in_array('persist', $value, true));
```

Comment:
Maybe could we use an array (`["persist"=>true]`) to allow passing more options later?

Reply:
I don't like the idea of adding an option "_just in case we need it later_"

Reply:
We're already adding an option :) It's important to add it in a future-proof way or we'll have to deprecate it and maintain a BC layer at some point, and it will hurt users as well as us.

Reply:
So should I support all those formats? `'persist'`, `['persist']`, `['persist' => true]`

Reply:
Only the array IMHO, to make the API simpler.

Reply:
But that would be weird if I use it in addition to `input` data: `['persist' => true, 'input']`